packages/module/src/AttachmentEdit/AttachmentEdit.test.tsx

```diff
@@ -51,4 +52,47 @@ describe('AttachmentEdit', () => {
     fireEvent.click(screen.getByText('Cancel'));
     expect(onCancelHandler).toHaveBeenCalled();
   });
+
+  it('should render custom button text for footer actions buttons', () => {
+    render(
+      <AttachmentEdit
+        code="Hello world"
+        fileName="greetings.txt"
+        isModalOpen={true}
+        onCancel={jest.fn()}
+        onSave={jest.fn()}
+        handleModalToggle={jest.fn()}
+        primaryActionButtonText="Save"
+        secondaryActionButtonText="Close"
+      />
+    );
+
+    screen.getByText('Save');
+    screen.getByText('Close');
+  });
+
+  it('should render AttachmentEdit with custom classNames', async () => {
+    render(
+      <AttachmentEdit
+        code="Hello world"
+        fileName="greetings.txt"
+        isModalOpen={true}
+        onCancel={jest.fn()}
+        onSave={jest.fn()}
+        handleModalToggle={jest.fn()}
+        primaryActionButtonText="Save"
+        secondaryActionButtonText="Close"
+        modalHeaderClassName="custom-header-class"
+        modalBodyClassName="custom-body-class"
+        modalFooterClassName="custom-footer-class"
+      ></AttachmentEdit>
+    );
+
+    const modalHeader = document.querySelector('header.custom-header-class');
+    expect(modalHeader).toBeInTheDocument();
+    const modalBody = document.querySelector('.custom-body-class');
+    expect(modalBody).toBeInTheDocument();
+    const modalfooter = document.querySelector('.custom-footer-class');
+    expect(modalfooter).toBeInTheDocument();
+  });
```

Comment:
Slightly more of a nit, but rather than grabbing and asserting this way in this file and the CodeModal test file, we could do something more along the lines of (would also involve importing `within` from testing-library/react:

```suggestion
    const modal = screen.getByRole('dialog');
    const modalHeader = within(modal).getByRole('banner');
    expect(modalHeader).toHaveClass('custom-header-class');
    const modalBody = modal.querySelector('#code-modal-body');
    expect(modalBody).toHaveClass('custom-body-class');
    const modalfooter = within(modal).getByRole('contentinfo');
    expect(modalfooter).toHaveClass('custom-footer-class');
```